vividus-plugin-mobile-app/src/main/resources/properties/profile/mobile_app/profile.properties

```diff
@@ -8,3 +8,6 @@ mobile-application.swipe.vertical.x-position=0
 
 mobile-environment.real-device=false
 mobile-environment.wda-bundle-id=
+
+appium.screenshot.shooting-strategy=mobile-app
+appium.screenshot.strategy.mobile-app.screenshot-shooting-strategy=SIMPLE
```

Comment:
I think we need to review the property naming at some point, but its's not necessary to be in this PR

Reply:
If you have any variants we can discuss them